src/tests.py

```diff
@@ -13,7 +13,7 @@ def test_actions():
     print(action_list)
     assert len(action_list) > 0, f"The action list should not be empty: {action_list}"
 
-    encoded_actions_prompt = baseline.get_encoded_action_list()
+    encoded_actions_prompt = baseline.get_encoded_input_prompt("")
```

Comment:
oh I see maybe we kinda test get_encoded_input_prompt here, except this is using a string instead of dummy inputs (and actually making sure that the prompt is what we expect it to be, if we feed it an Input.

Maybe make this an actual test by passing it a new Input(specify input params like name and url) and make sure the output is what we expected?

Reply:
+1 to creating a dummy input.

Reply:
Awesome, adding dummy `Input` objects in both function calls.